src/scss/components/ec-card/ec-card.story.js

```diff
@@ -15,6 +15,17 @@ stories
     </div>
       `,
   }))
+  .add('interactive card', () => ({
+    template: `
+    <div style="display: flex; height: 100vh; margin: 10px">
```

Comment:
wouldn't we want to have an interactive card with an `a` tag?

Reply:
@matallo The UX part does not have _text-decoration_ when we do hover or some other properties, so we would also need to cancel those properties that tag _a_ has predefined, anyway this could be to the tastes of consumers because it is a sass component and it is not a Vue component forcing a markup defined.